nixos/tests/blocky.nix

```diff
@@ -0,0 +1,53 @@
+import ./make-test-python.nix {
+  name = "blocky";
+
+  nodes = {
+    minimalConf = { ... }: {
+      services.blocky = {
+        enable = true;
+
+        settings = {
+          upstream = {
+            default = [ "8.8.8.8" "1.1.1.1" ];
+          };
+          port = 53;
+          httpPort = 4000;
+          logLevel = "info";
+        };
+      };
+    };
+
+    mixedConf = { ... }: {
```

Comment:
I don't understand what this is for. The description mentions merging configuration... there is no merge here.

Reply:
> ```yaml
> customDNS:
>   customTTL: 1h
>   mapping:
>     printer.lan: 192.168.178.3,2001:0db8:85a3:08d3:1319:8a2e:0370:7344
> ```
> 
> Then you could run `dig @127.0.0.1 printer.lan`, etc...

I'll try to do something like this. This is my first time doing network related packaging, thanks for pointing it out :sweat_smile: